src/extension/browserAction/images/well_done.svg

```diff
@@ -0,0 +1,85 @@
+<svg width="124" height="124" viewBox="0 0 124 124" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't see this SVG loaded into a page anywhere. I would guess it is supposed to go on the `confirmation.html` page, right?

Reply:
yea, ended up generating a .png instead. The SVG was a bit complex, so I was concerned with performance.